addon/doxywizard/config.h

```diff
@@ -18,14 +18,22 @@
 
 class Input;
 class QTextStream;
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
 class QTextCodec;
+#else
+class QStringEncoder;
+#endif
 
 bool parseConfig(
       const QString &fileName,
       const QHash<QString,Input *> &options
     );
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
I see a lot of these `#if` constructs being introduced. Wouldn't it be better to introduce one or two adapter functions that internally do the conditional switching and use these adapters at the places where there is now a `#if`?

Reply:
Indeed there quite a few `#if` constructs and it is not very flexible (e.g. when newer versions have again different requirements.
Adapter functions might be a good idea, though I'm not good in creating adapter functions.

Reply:
Ok, I'll give it a try. 

I also see some errors when launching the Qt6 build of doxywizard from the command line regarding missing signals:

```
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QComboBox::activated(const QString &)
qt.core.qobject.connect: QObject::connect: No such signal QButtonGroup::buttonClicked(int)
qt.core.qobject.connect: QObject::connect: No such signal QButtonGroup::buttonClicked(int)
qt.core.qobject.connect: QObject::connect: No such signal QButtonGroup::buttonClicked(int)
qt.core.qobject.connect: QObject::connect: No such signal QButtonGroup::buttonClicked(int)
qt.core.qobject.connect: QObject::connect: No such signal QButtonGroup::buttonClicked(int)
```

So I think some signals have been renamed or gotten a different signature with Qt6.

Reply:
Strange, I didn't get any message. Which version of Qt did you try? I used the Windows 6.1.2/msvc2019_64 version, though as this version is a dynamic version I needed to add `...\6.1.2\msvc2019_64\bin` to the `PATH`.

Reply:
I used 6.1.3, but I think you would only see these messages on Windows if you launch the executable with console attached (i.e. run `editbin /SUBSYSTEM:CONSOLE,6.00 /OSVERSION:5.1 doxywizard.exe`)

Reply:
Indeed after the `editbin` I do see the messages as well.

- Should we add the `editbin` to the creation of the `doxywizard.exe` on Windows?